### DISCUSSION TITLE
Welcome to neotype Discussions!

### DISCUSSION THREAD

**Discussion by kitlangton** [kitlangton/neotype#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sisamon** [comment]
HI, first thanks for this nice library, I was looking for something like this for my new project.
I used to program in Scala, but stopped some 5 years ago, now I am trying to get up to speed with all the changes in Scala 3.
Something I intend to test is using new type combined with squants, I still need to work out the details, and check how it works in terms of usability.
I will come back when I find out.
